test-cases/tests/documentation/template.md

```diff
@@ -0,0 +1,20 @@
+<Test Case Number> - <Jira Title>
```

Comment:
I think we need to use the i<number> format (e.g., i07: Verify Network Policy Settings in Admin Guide).